pkg/chaosdaemon/util.go

```diff
@@ -51,6 +51,7 @@ const (
 type ContainerRuntimeInfoClient interface {
 	GetPidFromContainerID(ctx context.Context, containerID string) (uint32, error)
 	ContainerKillByContainerID(ctx context.Context, containerID string) error
+	FormatContainerID(ctx context.Context, containerID string) (string, error)
```

Comment:
This is just a helper function, which shouldn't be part of the interface. Also it is too easy to be a function...

Reply:
Thank you for the review. Yes. It’s simple. However, I think it might be runtime related. So I put it into the interface. WDYT?

Reply:
It is runtime related, but it doesn't mean this should be a method in the interface, actually this function is too simple to be a function... Why do we need that? What's the problem of just using `containerID[len(dockerProtocolPrefix):]`? I think this is fine, we can totally remove the prefix check below
```
	if len(containerID) < len(dockerProtocolPrefix) {
		return "", fmt.Errorf("container id %s is not a docker container id", containerID)
	}
	if containerID[0:len(dockerProtocolPrefix)] != dockerProtocolPrefix {
		return "", fmt.Errorf("expected %s but got %s", dockerProtocolPrefix, containerID[0:len(dockerProtocolPrefix)])
	}
```

Reply:
`FormatContainerID` also is used to verify the ContianerID.